files/security_updates_checker.sh

```diff
@@ -0,0 +1,10 @@
+set -eo pipefail
+
+SECURITY_UPGRADES=$(apt-show-versions | grep upgradeable | grep security | awk -F':' '{print $1}')
```

Comment:
Could you add `apt-show-versions` to the list of packages to install? I don't think this usually ships with Ubuntu out of the box.

Reply:
@bdero Thanks, very good point! Just added it.